### PR TITLE
Update docker.md

### PR DESCRIPTION
**Pull request by julieschwartz18**
Removed Docker Image Creation > Via Command Line (testing only) section also general TW edits

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/docker.md [hunk 1]
```diff
@@ -2,31 +2,22 @@
...
 ## Why Use Docker?
-Primarily we use docker to run python scripts and integrations in a controlled environment. They run isolated from the server to prevent someone from accidentally damaging the server. By packaging libraries and dependencies together, we can prevent unknown issues from occurring since the environment is all the same.
-
+Primarily we use Docker to run Python scripts and integrations in a controlled environment. They run isolated from the server to prevent someone from accidentally damaging the server. By packaging libraries and dependencies together, we can prevent unknown issues from occurring since the environment is all the same.
 ## Script/Integration Configuration
-Specifying which docker image to use is done in the Cortex XSOAR IDE (Open: Settings -> Docker image name). If you don't specify a docker image, a default docker image using Python 2.7 is used. For new scripts and integrations, unless there is a specific reason to use Python 2 (for example: a need to use a library which is not available for Python 3), we require using a Python 3 image. 
-
```
Comment (anara123) [inline docs/integrations/docker.md:16]: need to revert the space between paragraph and note.

**Comment by julieschwartz18** [comment]
Thanks @anara123 ! I've made some additional edits, please check and let me know if it looks ok

**Comment by julieschwartz18** [comment]
Also removed the Advanced: Server - Container Communication section based on @anara123 's recommendation

File: docs/integrations/docker.md [hunk 2]
```diff
@@ -37,89 +29,26 @@ script:
...
+**Note:**    
+Other licenses may be permitted with specific approval.
+### Security Concerns
+It is imperative to perform due diligence on packages you choose to use. This includes verifying the package name is correct. In 2018 alone, a scan of PyPI resulted in the detection of 11 "typo-squatted" packages which were found to be malicious, as documented [here](https://medium.com/@bertusk/detecting-cyber-attacks-in-the-python-package-index-pypi-61ab2b585c67).
+### Docker Image Creation
+If you need to create a Docker image, proceed after having done due diligence and checking the licenses.
+### Docker File Repository (Required for Production)
+In most cases, if your integration is for public release, you need to push Docker files into the dockerfiles repository located [here](https://github.com/demisto/dockerfiles). Pushing into this repository will add the image (after an approval process) to the Docker hub Demisto organization. See the [README](https://github.com/demisto/dockerfiles/blob/master/README.md) file for instructions.  
 ```
```
Comment (glicht) [inline docs/integrations/docker.md:54]: Remove this line. It is creating a black code box at the end of the page.

**Comment by julieschwartz18** [comment]
TW edits complete.

**Review by anara123 — APPROVED** [review]

**Pull request merged**